math/src/field/fields/montgomery_backed_prime_fields.rs

```diff
@@ -257,6 +257,18 @@ where
     fn representative(x: &Self::BaseType) -> Self::RepresentativeType {
         MontgomeryAlgorithms::cios(x, &UnsignedInteger::from_u64(1), &M::MODULUS, &Self::MU)
     }
+
+    fn field_bit_size() -> usize {
+        let mut evaluated_bit = NUM_LIMBS * 64 - 1;
+        let max_element = M::MODULUS - UnsignedInteger::<NUM_LIMBS>::from_u128(1);
+        let one = UnsignedInteger::from_u128(1);
+
+        while ((max_element >> evaluated_bit) & one) != one {
+            evaluated_bit -= 1;
+        }
+
+        evaluated_bit + 1
```

Comment:
If you have access to limbs you can do the [checked_ilog2](https://doc.rust-lang.org/std/primitive.u64.html#method.checked_ilog2) method and operate on the result.

Reply:
That seems useful